js/ui/main.js

```diff
@@ -927,6 +927,8 @@ function notifyError(msg, details) {
  * extension system as well as debugging.
  */
 function _log(category, msg) {
+    if (category == undefined || msg == undefined) return;
```

Comment:
You could just use !category || !msg here because if they are null this won't catch it because you're only checking for undefined. As long as it is at least a variable, which parameters are, you don't need to check undefined.

In JS this checks

- null
- undefined
- NaN
- empty string ("")
- 0
- false